packages/debug/src/browser/debug-session-connection.ts

```diff
@@ -116,6 +113,7 @@ const standardDebugEvents = new Set<string>([
     'thread'
 ]);
 
+// TODO: Proper message RPC for debug session protocol
```

Comment:
I'm not sure this is really a todo: DAP is a JSON-based protocol, not rpc.

Reply:
Correct. This is a leftover comment that I forgot to remove. Good catch.